ONSdigital/eq-survey-runner
# Add cookie banner

### What is the context of this PR?
This PR is to add the cookie banner

### How to review 
- Open the app
- See that the cookie banner is displayed
- Open the 'application' tab in the inspector and click on cookies under storage and the url for the page.
- Check that "ons_cookie_policy" gets set with default consent i.e. all are set to true
- Click the "accept cookies" button
- Check it hides
- Refresh the page and check it's still hidden
- Look in the cookies in the inspector and check that "ons_cookie_message_displayed" is set
